x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/components/alert_counts.tsx

```diff
@@ -41,7 +41,7 @@ export const AlertCounts = ({ activeAlertCount, recoveredAlertCount, handleClick
       <AlertItem
         label={ACTIVE_NOW_LABEL}
         count={activeAlertCount}
-        color={activeAlertCount > 0 ? euiTheme.colors.dangerText : euiTheme.colors.successText}
+        color={activeAlertCount > 0 ? euiTheme.colors.dangerText : euiTheme.colors.textSuccess}
```

Comment:
should we also change `.dangerText` to `.textDanger`? VS code show `.dangerText` as deprecated.

Reply:
Nevermind, I just saw that you did that on your other PR 👍